python/whylogs/api/writer/whylabs.py

```diff
@@ -80,21 +81,27 @@ def option(
         if api_key is not None:
             self._api_key = api_key
 
-    def write(self, profile: Union[DatasetProfileView, DatasetProfile], dataset_id: Optional[str] = None) -> Any:
+    @deprecated_alias(profile="file")
+    def write(self, file: Union[DatasetProfileView, DatasetProfile], dataset_id: Optional[str] = None, **kwargs) -> Any:
```

Comment:
It looks like we can make this `file: Writable` and make `DatasetProfile <: Writable

Reply:
Can we pass `dataset_id` through `kwargs`?

Reply:
got it :)